src/Alert/index.js

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled, { css } from 'styled-components';
+
+import { Icon, Theme } from '..';
+import { Message } from './elements';
+import { STATUS_ICON_NAMES } from './constants';
+
+/**
+ * Alert
+ *
+ * This component is in charge of displaying
+ * an Alert for a user
+ *
+ * @param {string} className // className needed by styled components.
+ * @param {bool} closable // Whether it is possible to close the alert box.
+ * @param {string} message // An alert can have a message description.
+ * @param {func} onClose // An alert can have a clickable button to close it.
+ * @param {enum} type // The type of the message box.
+ *
+ * @return {jsx}
+ */
+
+const Alert = ({ className, closable, message, onClose, type }) => (
+  <div className={className} role="alert">
+    <Icon name={STATUS_ICON_NAMES[type]} color={Theme.palette.white} aria-hidden="true" />
+    <Message>{message}</Message>
+    {closable && (
+      <Icon
+        tabIndex="0"
+        role="button"
+        name="cross"
+        color={Theme.palette.white}
+        width="1.8rem"
+        height="1.8rem"
+        onClick={onClose}
+      />
+    )}
+  </div>
+);
+
+/**
+ * PropTypes Validation
+ */
+const { bool, func, oneOf, string } = PropTypes;
+Alert.propTypes = {
+  className: string,
+  closable: bool,
+  message: string.isRequired,
+  onClose: func,
+  type: oneOf(['success', 'info', 'warning', 'error']),
+};
+
+/**
+ * Default props
+ */
+Alert.defaultProps = {
+  className: '',
+  closable: false,
+  onClose: () => {},
+  type: 'info',
+};
+
+export default styled(Alert)`
+  display: flex;
+  align-items: center;
+
+  padding: 1.3rem;
+  width: 100%;
+
+  border-radius: 0.5rem;
+
+  color: ${({ theme: { palette } }) => palette.white};
+  background: ${({ theme: { palette } }) => palette.darkGrey};
+
+  cursor: pointer;
+
+  /* success */
```

Comment:
I know this is not what we've done till now, but I'm wondering whether adding a `getStatusBackColor` object in `constants.js` (or wherever) wouldn't be cleaner. It could have the following shape:

```javascript
const getStatusBackColor =({ palette }, type) => {
  return {
    info: palette.primary.default,
    /* ... */
  }[type];
};
```

and then, in the CSS part:

```javascript
background: ${getStatusBackColor(theme, type)};
```